scripts/generate-clients/copy-to-clients.js

```diff
@@ -25,6 +32,32 @@ const getOverwritablePredicate = packageName => pathName => {
   );
 };
 
+/**
+ * Copy the keys from newly-generated package.json to
+ * existing package.json. For each keys in new package.json
+ * we prefer the new key. Whereas for the values, we prefer
+ * the values in the existing package.json.
+ *
+ * This behavior enables as remove dependencies/scripts
+ * from codegen, but maintain the newer dependency versions
+ * in existing package.json
+ */
+const mergeManifest = (fromContent, toContent) => {
+  const merged = {};
+  const fromNames = Object.keys(fromConfig);
+  for (const name of fromNames) {
+    merged[name] = fromConfig[name];
```

Comment:
add this and remove else block
```suggestion
    // If key (say dependency) is present in both codegen and
    // package.json, we prefer latter
    merged[name] = toContent[name] || fromContent[name];
```